third_party/terraform/resources/resource_compute_instance_group_manager.go.erb

```diff
@@ -499,7 +499,7 @@ func resourceComputeInstanceGroupManagerRead(d *schema.ResourceData, meta interf
 	d.Set("description", manager.Description)
 	d.Set("project", project)
 	d.Set("target_size", manager.TargetSize)
-	if err = d.Set("target_pools", manager.TargetPools); err != nil {
+	if err = d.Set("target_pools", mapStringArr(manager.TargetPools, ConvertSelfLinkToV1)); err != nil {
```

Comment:
I'm assuming that somebody who has beta links in their config will still be ok because `target_pools` uses `selfLinkRelativePathHash` to hash

Reply:
yep! This only affects `ImportStateVerify`.